### ISSUE TITLE
new content not shown in index / toc

### ISSUE THREAD

**Issue by jmminy** [Simbul/baker#835]
Hi!
For some reasons, doubletap is not showing my new pagesin the toc or index.html.
Strangly it works using the browser, it's only in the app, the new elements are not showing. It stops at number 5.
Thanks in advance for your help

**Comment by folletto** [comment]
It's impossible to help you with this little information. Could you please go through the [Problems and Debugging](https://github.com/Simbul/baker/wiki/Problems-and-Debugging) page on the wiki and report back with the information? :)

If possible, a link to a `shelf.json` to test the problem it would be helpful for others to help you. :)

**Comment by jmminy** [comment]
Hi, in fact I went through this debug first and followed everything in the doc too. I'm running the app in standalone mode with the latest version of xcode and the current master of Baker (downloaded the 11th of this month). I have added new content to my app, till now it was working like a charm. The new elements declared in book.json and added to index.html and inc/toc.html do not display in the app (only the previous do). I have validated my json with the tool and there is no error. Testing the files in a browser shows the elements added. It's only in the app i don't see them. I have no error in the debug console and exporting just shows 4warnings (which i had before this issue)

**Comment by folletto** [comment]
Did you try all the steps in Problems and Debugging? Seems a problem related to caching that should go away if you just remove the app from the device/simulator, clean and rebuild.

**Comment by jmminy** [comment]
Yes of course and let me make you laugh, I did some CSS to display both bars: the top button (the one showing in the browser) and the one showing after double tap. Guess what, my new elements appear in the top menu but not in the other one. Believe me, index.html and inc/toc.html have the same content. This is strange. I also tried exporting to ios5.0 and above, same behavior the only difference is that using ios6 or 6.1 i have 23 issues and using ios5, i have only 4.
